packages/test/test-version-utils/src/versionUtils.ts

```diff
@@ -360,7 +376,7 @@ export function getRequestedVersion(
 		return baseVersion;
 	}
 	if (typeof requested === "string") {
-		return requested;
+		return resolveVersion(requested, false);
```

Comment:
by using resolveVersion, strings in formats like "^1.3.4" are normalized to "1.3.7" which is the version that is gonna be used anyway.